Changelog.md

```diff
@@ -1,3 +1,10 @@
+### Development
+
+Enhancements:
+
+* Expose the reporter used to run examples via example. (Jon Rowe, #1866)
```

Comment:
"via example" suggests that the reporter is returned from an `example`  method.  How about "via `RSpec::Core::Example#reporter`"?